pyuvdata/uvbeam.py

```diff
@@ -1084,6 +1084,60 @@ def __iadd__(self, other):
         self.__add__(other, inplace=True)
         return self
 
+    def _stokes_matrix(self, n):
+        """
+        Calculates Pauli matrices (where indices are reordered from the quantum mechanical convention
+        to an order which gives the ordering of the Stoes vector as [I, Q, U, V]) according to
+        https://arxiv.org/pdf/1401.2095.pdf.
+
+        Args :
+            n : Index for which to calculate the Pauli matrix; n should lie between 0 and 3 for the 4x4
```

Comment:
We prefer not to use single letter variable names. Please pick something that has more meaning. I don't know reading this what 'n' is supposed to represent: is it a polarization index?

Reply:
done!